website/docs/r/application_gateway.html.markdown

```diff
@@ -267,7 +269,7 @@ A `path_rule` block supports the following:
 
 A `probe` block support the following:
 
-* `host` - (Required) The Hostname used for this Probe. If the Application Gateway is configured for a single site, by default the Host name should be specified as ‘127.0.0.1’, unless otherwise configured in custom probe.
+* `host` - (Required) The Hostname used for this Probe. If the Application Gateway is configured for a single site, by default the Host name should be specified as ‘127.0.0.1’, unless otherwise configured in custom probe. Cannot be set if `pick_host_name_from_backend_http_settings` is set to true.

```

Comment:
Could we change the `host` to be optional? We also prefer `true` to be enquoted.
```suggestion
* `host` - (Optional) The Hostname used for this Probe. If the Application Gateway is configured for a single site, by default the Host name should be specified as ‘127.0.0.1’, unless otherwise configured in custom probe. Cannot be set if `pick_host_name_from_backend_http_settings` is set to `true`.
```

Reply:
done. good spot.